src/test/debuginfo/type-names.rs

```diff
@@ -1,208 +1,268 @@
 // ignore-lldb
-// ignore-gdb // Test temporarily ignored due to debuginfo tests being disabled, see PR 47155
+
+// GDB changed the way that it formatted Foreign types
+// min-gdb-version: 9.2
 
 // compile-flags:-g
 
+// === GDB TESTS ===================================================================================
+
 // gdb-command:run
 
 // STRUCTS
 // gdb-command:whatis simple_struct
-// gdbg-check:type = struct Struct1
-// gdbr-check:type = type_names::Struct1
+// gdb-check:type = type_names::Struct1
```

Comment:
I noticed that you removed the `gdbg-` / `gdbr-` distinction in this file. We can give it a try but we might run into problems like the one we just had on Android (but hopefully not because the android GDB version must be much older than 9.2 for it not have native Rust support)

Reply:
Looks like this test was ignored for Android since the GDB version was too old